js/a11y/browserFocus.js

```diff
@@ -45,19 +45,32 @@ export default class BrowserFocus extends Backbone.Controller {
     if (event.target !== event.currentTarget) {
       return;
     }
-    // Do not auto next if the focus isn't returning to the body
-    if (!$(event.relatedTarget).is('body')) {
+    // Do not auto next if the focus isn't returning to the body or html element
+    // or if we're not losing focus
+    if (!$(event.relatedTarget).is('body, html') && event.relatedTarget !== null) {
       return;
     }
     // Check if element losing focus is losing focus
-    // due to the addition of a disabled class
-    if (!$element.is('[disabled]') && $element.css('display') !== 'none' && $element.css('visibility') !== 'hidden') {
+    // due to the addition of a disabled class, display none, visibility hidden,
+    // or because it has been removed from the dom
+    if (!$element.is('[disabled]') && $element.css('display') !== 'none' && $element.css('visibility') !== 'hidden' && $element.parents('html').length) {
```

Comment:
This line is really long and complex. Suggest using more [descriptive conditions](https://github.com/adaptlearning/documentation/blob/master/01_cross_workstream/style_guide.md#use-descriptive-conditions).